tests/FunctionalTests/CaseAddTagsTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dotfile\Tests\FunctionalTests;
+
+use Dotfile\DotfileClient;
+use Dotfile\Model\Case\CaseTags;
+use Dotfile\Model\Case\CaseTagsInput;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\HttpClient\Response\MockResponse;
+
+class CaseAddTagsTest extends TestCase
+{
+    public function testCaseCreateWithMinimumValidData(): void
+    {
+        $client = new DotfileClient(new MockHttpClient([
+            new MockResponse((string) file_get_contents(__DIR__.'/../fixtures/case_add_tags_with_minimal_data_response.json')),
+        ]));
+
+        $input = new CaseTagsInput();
+        $input->caseId = '39cbd6d5-4da5-4d94-ae71-84895c5e552a';
+        $input->tags = ['A faire'];
+
+        $caseAddTags = $client->tags->create($input, $input->caseId);
+
+        $this->assertInstanceOf(CaseTags::class, $caseAddTags);
+        $this->assertSame([
+            'id' => 'ea5c5935-2ec8-4519-a526-7ab25c71ac4e',
+            'label' => 'A faire'], $caseAddTags->tags[0]);
```

Comment:
```suggestion
        $this->assertSame([
            'id' => 'ea5c5935-2ec8-4519-a526-7ab25c71ac4e',
            'label' => 'A faire'
        ], $caseAddTags->tags[0]);
```

Une petite modification pour le formattage